src/goto-instrument/goto_program2code.cpp

```diff
@@ -178,12 +178,12 @@ goto_programt::const_targett goto_program2codet::convert_instruction(
     case END_FUNCTION:
     case DEAD:
       // ignore for now
-      dest.copy_to_operands(code_skipt());
+      dest.add(code_skipt());
       return target;
 
     case FUNCTION_CALL:
     case OTHER:
-      dest.copy_to_operands(target->code);
+      dest.add(target->code);
```

Comment:
I am surprised there is still a use of `target->code`, shouldn't this be `target->get_other()`?

Reply:
Indeed, I've added a commit to fix this.

Reply:
Thank you!!